microsoft-edge/extensions-chromium/publish/auto-update.md

```diff
@@ -22,6 +22,19 @@ ms.date: 11/04/2022
    See the License for the specific language governing permissions and
    limitations under the License.  -->
 # Automatically update an extension in Microsoft Edge
+<!-- alts:
+# Autoupdate
+# Set an extension to automatically update
```

Comment:
I much prefer this alternative title to be honest.
The problem I have with the current title is that it is too similar to the title of this other article: "Update a Microsoft Edge extension".

Compare:
- "Update a Microsoft Edge extension"
- "Automatically update an extension in Microsoft Edge"

The former is for developers to update their extension in partner center.
The latter is for users of an extension to automatically get new versions of that extension in their browser.

Let's go with "Set an extension to automatically update". Do you agree?

Reply:
changed to "Set an extension to automatically update" in all usual spots